src/Microsoft.Extensions.Configuration.Json/JsonConfigurationFileParser.cs

```diff
@@ -87,15 +88,15 @@ private void VisitArray(JArray array)
             }
         }
 
-        private void VisitPrimitive(JToken data)
+        private void VisitPrimitive(JValue data)
         {
             var key = _currentPath;
 
             if (_data.ContainsKey(key))
             {
                 throw new FormatException(Resources.FormatError_KeyIsDuplicated(key));
             }
-            _data[key] = data.ToString();
+            _data[key] = data.ToString(CultureInfo.InvariantCulture);
```

Comment:
Curious, is there any functional or perf difference between this fix and just switching this line to use `data.Value<string>()`? I am not against the way it is done here if we think the contract of `.Value<string>()` is weak re culture.